src/pages/General.vue

```diff
@@ -34,6 +38,7 @@ export default {
     return {
       /** @type Settings */
       settings: ipcRenderer.sendSync('settings-read'),
+      version:  ipcRenderer.sendSync('get-app-version'),
```

Comment:
While this is fine, it'd probably better to do things asynchronously (to not block rendering the UI). Having the version fill in after the page loads shouldn't be an issue.